lib/chutney/TorNet.py

```diff
@@ -2542,11 +2542,10 @@ def ConfigureNodes(nodelist):
 def getTests():
     tests = []
     chutney_path = get_absolute_chutney_path()
-    if len(chutney_path) > 0 and chutney_path[-1] != '/':
-        chutney_path += "/"
-    for x in os.listdir(chutney_path + "scripts/chutney_tests/"):
-        if not x.startswith("_") and os.path.splitext(x)[1] == ".py":
-            tests.append(os.path.splitext(x)[0])
+    chutney_tests_path = chutney_path / "scripts" / "chutney_tests"
+    for x in chutney_tests_path.glob("*.py"):
+        if not x.startswith("_"):
```

Comment:
Will this work?  I don't think a path object has a startswith() method.  If it did, it would refer to the  path as a whole, when the previous code is looking at whether there is an underscore at the start of the filename component.

Reply:
See bc7abead8e2be29bc89d485cb86c01b92c5165b1